survival_problem_grad_descent.py

```diff
@@ -168,11 +168,13 @@ def _get_parallel_sum(worker_list, shared_obj):
         sorted_sample_labels = sorted(list(set(self.sample_labels)))
         log.info("Obtained gradients %s" % (time.time() - st))
 
-        # Get the expected scores
+        # Get the expected scores and their sum
         expected_scores = {label: 0 for label in sorted_sample_labels}
+        expected_scores_sum = 0
```

Comment:
I find it a bit surprising that you don't parallelize calculating the sum of the expected scores but you do parallelize calculating the sum of the expected score * expected score^T.
Is the latter a lot slower such that it requires parallelization?

Reply:
I haven't tested this. I was modeling it after `ScoreScoreWorker` and L174. I'll test it and see what comes out.